cloudformation/external-slurmdbd/external_slurmdbd/external_slurmdbd_stack.py

```diff
@@ -239,10 +240,10 @@ def _add_slurmdbd_accounting_security_groups(self):
             description="Allow Slurm accounting traffic from the cluster head node",
         )
 
-        slurmdbd_client_sg.add_egress_rule(
+        slurmdbd_client_sg.add_ingress_rule(
             peer=slurmdbd_server_sg,
-            connection=ec2.Port.tcp(6819),
-            description="Allow Slurm accounting traffic to the slurmdbd instance",
+            connection=ec2.Port.tcp_range(6820, 6829),
```

Comment:
The port can be configured in `slurm.conf`  ..if we hardcode here the port range we may impact customers that use different ports (i.e. `CustomSlurmSettings`).

Possibly we should retrieve these values from the config ..or at least document this limitation.

Reply:
The `slurm.conf` won't be available on the slurmdbd stack. The only thing we can do is to make the port range configurable via a CFN parameter: I put a TODO in the code, but maybe we want it to expose it immediately?

Reply:
> The slurm.conf won't be available on the slurmdbd stack.
Yes .. it would be better to have a parameter for this to ensure that if customers use a different port range in the config they can have a simple way to align the slurmdbd stack.

Reply:
We discussed about this and decided not to include the customization of the port ranges for the time being (users would have other mechanisms to do so, such as editing the template or changing the ingress rule by hand via the EC2 console).